source/catalog.html

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+<html class="page" lang="ru">
+  <head>
+    <meta charset="utf-8">
+    <title>HTML Academy: Каталог Седона</title>
+  </head>
+  <body class="page_ _body">
```

Comment:
У тебя не корректно написанные классы. По БЕМ будет так
```suggestion
  <body class="page__body">
```
Должно быть слитно. Это по всем проекту, так что отмечу тут